bin/ytdl.js

```diff
@@ -332,7 +333,9 @@ if (opts.infoJson) {
 
       readStream.on('data', (data) => {
         dataRead += data.length;
-        updateProgress();
+        if (is_stdout_mutable) {
```

Comment:
Would prefer to put this check above, on the same line where `liveBroadcast` is also checked, to avoid even creating the `updateProgress` function or requiring `lodash.throttle`